client-js/queryEditor/SchemaSidebar.js

```diff
@@ -335,20 +336,38 @@ class SchemaInfoColumnItem extends React.Component {
         )
       }
     }
-    return (
-      <li className="list">
-        <span
-          onMouseOver={this.handleMouseOver}
-          onMouseOut={this.handleMouseOut}
-          className="dib"
-          style={{ minWidth: '230px' }}
-        >
-          {column_name}
-          <span className="silver"> ({data_type})</span>
-          {getCopyToClipboard()}
-        </span>
-      </li>
-    )
+    if (col_desc) {
+      return (
+        <li className="list">
```

Comment:
Instead of having the full `<li>` node duplicated, could you refactor to put the description into a variable and then reference that in a single `<li>`?

perhaps something like 

```js
const description = col_desc && ` - ${col_desc}`;

return (
  <li>
    {/* all the other stuff */}
    <span className="silver"> ({data_type})</span>{description}{getCopyToClipboard()}
  </li>
)
```